plasma-sdk/apps/bill-split/src/app/globals.css

```
@@ -1,210 +1,610 @@
 @tailwind utilities;
 
-/* ============================================================================
-   CSS Variables
-   ============================================================================ */
+/* ========================================
+   SPLITZY CLAYMORPHISM THEME
+   Brand Color: Teal (#14B8A6)
+   Soft, friendly, tactile bill-splitting UI
+   ======================================== */
+
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=Inter:wght@400;500;600&display=swap');
```

Biome (2.1.2)

[error] This `@import` is in the wrong position.

Any `@import` rules must precede all other valid at-rules and style rules in a stylesheet (ignoring `@charset` and `@layer`), or else the `@import` rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)